### DISCUSSION TITLE
Do the judgments of merge and first-time contributors have a bug?

### DISCUSSION THREAD

**Discussion by kenji-miyake** [community/community#29757] [Pull Requests]
This discussion was deleted by request from @myoung34.

-> See https://marcyoung.us/post/zuckerpunch/ and https://github.blog/changelog/2022-10-12-reverted-recent-change-that-caused-some-pull-requests-to-be-incorrectly-marked-as-merged/.

**Comment by myoung34** [comment]


**Reply by kenji-miyake** [reply]
@myoung34 Thank you for your quick reply. :smile: Roughly when can it be public?
By the way, as I sent you an email, I'd appreciate it if you could personally tell me the details.
